### PR TITLE
base: active header tab designation

### LINKED ISSUE #99: ui: show active tab
ui-show-active-tab

### PR DESCRIPTION
**Pull request by bouzlibop**
* Adds highlight to an active header tab. (closes #99)

Signed-off-by: Adrian Pawel Baran <adrian.pawel.baran@cern.ch>

### CODE REVIEW

File: zenodo/base/views.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def register_menu_items():
 
     item = current_menu.submenu('main.browse')
     item.register('', _('Browse'), order=3)
+    item._active_when = lambda: request.endpoint.startswith("search.collection")
```
Comment (bouzlibop) [inline zenodo/base/views.py]: @lnielsen, could you take a look ? I think accessing the `private` attribute in this way is really bad practise. Is there any better way to do it?

Comment (lnielsen) [inline zenodo/base/views.py]: You should be able to provide in  ``item.register(active_when=...)`` (see https://github.com/inveniosoftware/flask-menu/blob/master/flask_menu/__init__.py#L75)

Comment (bouzlibop) [inline zenodo/base/views.py]: Sorry, that was a huge oversight :see_no_evil:

**Comment by lnielsen** [comment]
Merge via 78e184f428b1889a5771bf9f15e3ff8cd696a979

**Pull request closed without merging**